tests/brokers/rabbit/test_connect.py

```diff
@@ -58,3 +58,19 @@ async def test_connect_merge_args_and_kwargs_native(self, settings):
         broker = self.broker("fake-url")  # will be ignored
         assert await broker.connect(url=settings.url)
         await broker.close()
+
+    @pytest.mark.asyncio()
+    async def test_connection_alive(self, settings):
```

Comment:
Why are you duplicate these tests in the inheritor? They will be runned already due parent declaration